packages/components/avatar/src/avatar.tsx

```diff
@@ -61,6 +65,11 @@ export const Avatar = forwardRef<AvatarProps, "span">((props, ref) => {
     ...rest
   } = omitThemingProps(props)
 
+  const onLoad: UseImageProps["onLoad"] = (e) => {
+    runIfFn(onLoad, e)
+    setIsLoaded(true)
+  }
+
```

Comment:
You can make this more concise by passing it directly to the onLoad prop

```suggestion
  const onLoad = callAllHandlers(onLoad, () => {
    setIsLoaded(true)
  })
```